upi/openstack/inventory.yaml

```diff
@@ -28,11 +28,10 @@ all:
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
 
-      # The public network, if available. Required for os_api_fip and
-      # os_ingress_fip.
-      # If non-empty, an additional floating IP will be created and attached to
-      # the bootstrap machine to allow the retrieval of a log bundle in case of
-      # install failure ('must-gather').
+      # The public network providing connectivity to the cluster. If not
+      # provided, the cluster must be provided connectivity another way.
```

Comment:
perhaps this could be rephrased to: "the cluster external connectivity must be provided in another way"?